state-chain/runtime/src/lib.rs

```diff
@@ -872,7 +872,7 @@ impl pallet_cf_broadcast::Config<Instance2> for Runtime {
 	type EnsureThresholdSigned =
 		pallet_cf_threshold_signature::EnsureThresholdSigned<Self, PolkadotInstance>;
 	type BroadcastReadyProvider = BroadcastReadyProvider;
-	type BroadcastTimeout = ConstU32<{ 10 * MINUTES }>;
+	type BroadcastTimeout = ConstU32<{ 4 * MINUTES }>;
```

Comment:
Polkadot? I assume is supposed to instance 4 for arb?

Reply:
I figured there was no reason not to reduce it for polkadot too - 4 is a bit of arbitrary choice, but 10 minutes for finality seemed excessive.